src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -444,8 +444,19 @@ impl AstDisplay for Envelope {
                 f.write_str("DEBEZIUM");
                 f.write_node(mode);
             }
-            Self::Upsert => {
+            Envelope::Upsert { order_by } => {
                 f.write_str("UPSERT");
+
+                if !order_by.is_empty() {
+                    f.write_str("(");
```

Comment:
you need a space before here right? the ddl tests show this

Reply:
Yes, let me add that!

Reply:
Done!